wgpu-core/src/device/mod.rs

```diff
@@ -998,15 +998,31 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
         let (format, view_kind, range) = match desc {
             Some(desc) => {
                 let kind = conv::map_texture_view_dimension(desc.dimension);
+                let level_count = desc.base_mip_level + desc.level_count;
```

Comment:
the `level_count` is confusing here, since it's not counting either the levels of the view, or the levels of the parent.
Maybe `required_level_count`?